Update save info on stage change

EJ125 opened the pull request.

EJ125 committed:
Update save info on stage change

fig02 commented on external/libtww/include/d/com/d_com_inf_game.h:
```diff
@@ -1608,6 +1608,8 @@ inline u8 dComIfGs_getChartDeciphered() {
     return g_dComIfG_gameInfo.info.getPlayer().mMap.getChartDeciphered();
 }
 
+LIBTWW_DEFINE_FUNC(dComIfGs_setGameStartStage__Fv,void, setGameStartStage, (void));
+
```
In this case it would be good to just make the function name the real function name from the game, so `dComIfGs_setGameStartStage`

EJ125 replied:
Made the correction. Thanks.

EJ125 committed:
correcting function name to match real name

EJ125 committed:
Couple other tweaks

block certain text from being drawn during load, rewrote the scene info function a bit

EJ125 committed:
Something i forgot to remove when trying stuff

EJ125 committed:
tidying up position of this menu
